### DISCUSSION TITLE
Azure GitHub Action is currently creating extra directories

### DISCUSSION THREAD

**Discussion by fiveisprime** [community/community#25024] [Actions]
<p>Tried to setup a new action today for deployment to Azure and the following file was created:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e7af3b1562d4cb9c443099622c02e97e3e56b09a.png" data-download-href="/uploads/short-url/x3zJ0renXuL15n36mX2CEtbMA66.png?dl=1" title="Annotation 2020-01-29 171932.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1186948/181088472-70c51d39-829b-4696-a8ca-0c7ddb9d9d42.png" alt="Annotation 2020-01-29 171932.png" data-base62-sha1="x3zJ0renXuL15n36mX2CEtbMA66" width="690" height="232" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e7af3b1562d4cb9c443099622c02e97e3e56b09a_2_690x232.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e7af3b1562d4cb9c443099622c02e97e3e56b09a_2_1035x348.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/e/e7af3b1562d4cb9c443099622c02e97e3e56b09a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e7af3b1562d4cb9c443099622c02e97e3e56b09a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2020-01-29 171932.png</span><span class="informations">1272×428 35.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This is not just forks, I created a fresh repo and saw the same result.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>I checked your <a href="https://github.com/fiveisprime/StickerApp" rel="nofollow noopener">repo</a> but not found related yaml file, even no workflow history. Where is your azure.yaml file? And is the issue gone now?</p>
<p>I forked your repo and yaml file can be created without extra directories. If issue persists for you, please change to another browser for a try.</p>

**Comment by fiveisprime** [comment]
<p>I had force pushed over it, which is why you didn’t see the file. I was able to recreate this in several repos, but it appears to be fixed. :tada:</p>
